internal/testdata/size_enum_encoding.go

```diff
@@ -1,39 +0,0 @@
-// Code generated by go-enum-encoding DO NOT EDIT
```

Comment:
why this file was removed? can you keep it?

Reply:
I consider the files in `internal/testdata/exp` the source of truth for the test, while the output files in `internal/testdata` are generated by the running tests and should be temporary. In fact, we should clean them up before/after the test - otherwise a test that failed to generate anything and also failed to report an error - will pass. What do you think?

Specifically for `internal/testdata/size.go`, it looks like it is/was not being used for testing to begin with. Maybe left out unintentionally?

Reply:
ok. all looks good. please don't delete files. 

I hear you point, it makes sense. let's move those to tests later. and for now better to keep those.